learningequality/studio
# Add function for indexedDB clearing

## Description

* Adds a function to clear all of our indexedDB tables
* Fixes error in channel deletion mutation name

## Steps to Test

- Type `window.resetDB()` in the console
- Confirm that your indexedDB is now gone
- Refresh the page after a few seconds to confirm that the deletions have not been synced to the backend
